manifests/java.yml

```
@@ -446,7 +446,31 @@ tests/:
           '*': v1.39.0
           spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
           spring-boot-payara: bug (produces 500 errors)
-      test_ssrf.py: missing_feature
+      test_ssrf.py:
+        Test_Ssrf_BodyJson:
+          '*': irrelevant
```

```suggestion
          '*': missing_feature (missing endpoint)
```

Let's not use `irrelevant`, since that will skip the test.